js/content.js

```diff
@@ -39,7 +39,7 @@ let pastNum = null;
 let isZoomEnabled = false;
 
 //set to create a max interval between notification alerts
-let time = Date.now();
+let lastSetTime = Date.now();
```

Comment:
Name still isn't descriptive enough, something like `lastNotificationTime` is good